src/tapis-app/src/Sections/Apps/Apps.tsx

```diff
@@ -33,12 +34,12 @@ const Apps: React.FC = () => {
       <ListSectionHeader>Apps</ListSectionHeader>
       <ListSectionBody>
         <ListSectionList>
-          <AppsListing onSelect={appSelectCallback} />
+          <AppsListing onSelect={appSelectCallback} select="jobAttributes,version" />
         </ListSectionList>
         <ListSectionDetail>
           <ListSectionHeader type={"sub-header"}>Launcher</ListSectionHeader>
```

Comment:
Do you think we should we call this the "Job Launcher" too?
```suggestion
          <ListSectionHeader type={"sub-header"}>Job Launcher</ListSectionHeader>
```